src/components/chat/commented-documents.tsx

```diff
@@ -196,24 +211,37 @@ export const MyWorkDocuments: React.FC<JProps> = ({doc, index, sectionDoc, handl
       navTab = ENavTab.kClassWork;
     }
   }
-  const title =  useDocumentCaption(sectionDoc as DocumentModelType);
+  const title =  useDocumentCaption(sectionOrNetworkDoc as DocumentModelType);
+  //to do : add the download hook to download network docs
+  // add yellow div behind the svg
```

Comment:
Is this comment still relevant? 
If so, change `to do :` to `TODO:` that will make it consistent with other todos in the code so we can search for all of them.